package.json

```diff
@@ -18,41 +18,52 @@
     }
   ],
   "scripts": {
-    "build": "bower install && gulp",
-    "jshint": "gulp jshint",
-    "jscs": "jscs gulpfile.js assets/scripts/*.js"
+    "build:production": "export SAGE_ENV=production && webpack",
+    "build": "export SAGE_ENV=development && webpack",
+    "watch": "export SAGE_ENV=development && node dev.js"
```

Comment:
The exports can be removed. They aren't needed when defining an env var
inline like that.

On Tuesday, 15 March 2016, QWp6t <notifications@github.com> wrote:

> In package.json
> <https://github.com/roots/sage/pull/1590#discussion_r56259232>:
>
> > @@ -18,41 +18,52 @@
> >      }
> >    ],
> >    "scripts": {
> > -    "build": "bower install && gulp",
> > -    "jshint": "gulp jshint",
> > -    "jscs": "jscs gulpfile.js assets/scripts/*.js"
> > +    "build:production": "export SAGE_ENV=production && webpack",
> > +    "build": "export SAGE_ENV=development && webpack",
> > +    "watch": "export SAGE_ENV=development && node dev.js"
>
> I'm pretty sure none of these will work in Windows. Users will just get an
> error saying that it can't find the command export, or worse they might
> have an export command, but it does something completely different and
> yields wildly unexpected results.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/roots/sage/pull/1590/files/82cd4c32f6710fa5673eb6c6ce3386decfd255a2#r56259232>
>